src/provider_manifests.py

```diff
@@ -90,17 +98,13 @@ def __init__(self, charm, charm_config, kube_control, integrator):
     @property
     def config(self) -> Dict:
         """Returns current config available from charm config and joined relations."""
-        config: Dict = {}
-
-        if self.kube_control.is_ready:
-            config["image-registry"] = self.kube_control.get_registry_location()
-            config["cluster-name"] = self.kube_control.get_cluster_tag()
-
-        if self.integrator.is_ready:
-            config["cloud-conf"] = self.integrator.cloud_conf_b64
-            config["endpoint-ca-cert"] = self.integrator.endpoint_tls_ca
-
-        config.update(**self.charm_config.available_data)
+        config = {
+            "image-registry": self.kube_control.get_registry_location(),
+            "cluster-name": self.kube_control.get_cluster_tag(),
```

Comment:
these methods: `get_registry_location` and `get_cluster_tag` already check is_ready. They return `None` if it's not